editor/editor_plugin.cpp

```diff
@@ -450,7 +450,7 @@ void EditorPlugin::_bind_methods() {
 	gizmo.return_val.hint = PROPERTY_HINT_RESOURCE_TYPE;
 	gizmo.return_val.hint_string = "EditorSpatialGizmo";
 	ClassDB::add_virtual_method(get_class_static(), gizmo);
-	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "get_name"));
+	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "_get_name"));
```

Comment:
Actually since no other node has `_get_name method` but each one has `get_name` I don't find this very consistent, maybe as a @vnen suggested `get_plugin_name` should be considered?